src/Apps/Artwork/Components/ArtworkRelatedArtists.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Flex } from "@artsy/palette"
+import { ContextConsumer } from "Artsy/Router"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ArtistCardFragmentContainer as ArtistCard } from "Styleguide/Components"
+import { Header } from "./OtherWorks/Header"
+
+import { ArtworkRelatedArtists_artwork } from "__generated__/ArtworkRelatedArtists_artwork.graphql"
+
+export interface ArtworkRelatedArtistsProps {
+  artwork: ArtworkRelatedArtists_artwork
+}
+
+export class ArtworkRelatedArtists extends React.Component<
+  ArtworkRelatedArtistsProps
+> {
+  render() {
```

Comment:
This component should probably check the length of the related edges, and cleanly not render the rail if there aren't any related results (there might not be for reasons).

Reply:
👍 Will add!